check-before-deployment.md

```diff
@@ -189,6 +189,22 @@ To check whether the NTP service is installed and whether it synchronizes with t
     Active: active (running) since 一 2017-12-18 13:13:19 CST; 3s ago
     ```
 
+    - If it returns `Unit ntpd.service could not be found.` then try the following:
```

Comment:
This tells users what this command is about before they try it.
```suggestion
    If it returns `Unit ntpd.service could not be found.`, then try the following command, which configures your system to use `chronyd` instead of `ntpd` to perform clock synchronization with NTP:
```